app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -0,0 +1,116 @@
+// /*
+//  * This Source Code Form is subject to the terms of the Mozilla Public License,
+//  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

Reply:
I have looked at it and removed the uneccessary comments

Reply:
I do not see any changes.

Reply:
I removed all the unnecessary lines of comments.